### DISCUSSION TITLE
Adding to an existing github pages

### DISCUSSION THREAD

**Discussion by dosweiler** [osmoscraft/osmosfeed#84] [Q&A]
Hi, I'm trying to add the osmofeed to an already existing GitHub pages and I'm having trouble getting to update the feed. Is there any advice for adding it to an existing one?

**Comment by chuanqisun** [comment]
I assume you want to merge the output of osmosfeed with the existing files in the GitHub pages directory and you might be running into trouble because osmosfeed cleans up the directory for each new build.

- You can use [`keep_files`](peaceiris/actions-gh-pages@v3) option in the workflow setup to merge with existing files.
- You can use [`destination_dir`](https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-deploy-to-subdirectory-destination_dir) to deploy into a subdirectory.
